### PR TITLE
Code Quality: Added license header to C# phase2

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Add license header

```cs
// Copyright (c) 2023 Files Community
// Licensed under the MIT License. See the LICENSE.
```

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
  - Related #11853 
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

**Comment by 0x5bfa** [comment]
Changed C# files:

- **
- ValueConverters/**
- ViewModels/**
- Views/**

File: src/Files.App/Files.App.csproj [hunk 1]
```diff
@@ -71,6 +71,7 @@
     </ItemGroup>
     <ItemGroup>
       <None Remove="UserControls\OpacityIcon.xaml" />
+      <None Remove="Views\Settings\s5lspshi.5ja~" />
```
Comment (yaira2) [inline src/Files.App/Files.App.csproj:74]: What's this?

Comment (0x5bfa) [reply src/Files.App/Files.App.csproj:74]: Don't know as well. I'll fix soon.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**